auth/options.go

```diff
@@ -112,17 +121,59 @@ func NewGenerateOptions(opts ...GenerateOption) GenerateOptions {
 	return options
 }
 
+type LoginOptions struct {
```

Comment:
Remove LoginOptions considering we're just going to have the Token method?

Reply:
Thanks, missed this